libraries/AP_Winch/AP_Winch_Backend.h

```diff
@@ -36,6 +36,9 @@ class AP_Winch_Backend {
     // returns current length of line deployed
     virtual float get_current_length() const = 0;
 
+    // returns current line tension
+    virtual uint16_t get_current_tension() const = 0;
```

Comment:
Should probably return the tension in a refrence argument and return bool if it has a number that it filled in. If you add a defualt function that just returns false you won't have to add the method to the PWM backend which doesn't do anything.